app/assets/stylesheets/master.css.scss

```diff
@@ -0,0 +1,20 @@
+#chessboard {
+    width: 560px;
+    height: 560px;
+    border: 3px solid #000;
+    position: fixed;
+}
+
+.white, .black {
+    width: 70px;
+    height: 70px;
+    float: left;
```

Comment:
`float` has a weird side effect. We need to use a common hax called `clearfix`.

Bourbon has a clearfix add on:

http://bourbon.io/docs/#clearfix

you can learn more about clearfix:
http://learnlayout.com/clearfix.html